src/analysis/normed_space/spectrum.lean

```diff
@@ -93,8 +102,9 @@ by { refine bsupr_le (λ k hk, _), exact_mod_cast norm_le_norm_of_mem hk }
 
 open ennreal polynomial
 
+variable (𝕜)
 theorem spectral_radius_le_pow_nnnorm_pow_one_div (a : A) (n : ℕ) :
-  spectral_radius 𝕜 a ≤ ∥a ^ (n + 1)∥₊ ^ (1 / (n + 1) : ℝ) :=
+  spectral_radius 𝕜 a ≤ (∥a ^ (n + 1)∥₊ ^ (1 / (n + 1) : ℝ) : ℝ≥0∞) :=
```

Comment:
Was there a particular reason to add the explicit coercion? I think it looks nicer without.

Reply:
At one point I was in coercion hell with the `rpow` and the cast from `nnreal` to `ennreal`, so I think I added it when trying to figure that out, but you're right, it should just be removed.